packages/fxa-auth-server/lib/senders/emails/partials/button/index.ts

```diff
@@ -6,7 +6,7 @@ export const button = `
   <mj-include path="./lib/senders/emails/css/button/index.css" type="css" css-inline="inline" />
   <mj-section>
     <mj-column>
-      <mj-button css-class="primary-button" href="<%= baseURL %><%= anotherDeviceURL %>"><%= buttonText %></mj-button>
+      <mj-button css-class="primary-button" href="<%= baseUrl %><%= anotherDeviceUrl %>"><%= buttonText %></mj-button>
```

Comment:
What if `anotherDeviceUrl` and other URLs incorporated `baseUrl` on the server side, so we don't need to pass down `baseUrl` as a variable everywhere?